jscomp/main/jsoo_main.ml

```diff
@@ -96,6 +96,7 @@ let implementation ~use_super_errors prefix impl str  : Js.Unsafe.obj =
   try
   impl (Lexing.from_string
     (if prefix then "[@@@bs.config{no_export}]\n#1 \"repl.ml\"\n"  ^ str else str ))
+  |> Reactjs_jsx_ppx_v3.rewrite
```

Comment:
Would you like this to only be run conditionally?

Reply:
Currently the playground on the Reason site only compiles conditionally.  Are there plans to change it to always compile?  Reason JSX vs ReasonReact JSX might be something to consider.

Reply:
Yeah talking to Bob offline and we will probably compile two different versions - one with this built in.